### DISCUSSION TITLE
How to use it??

### DISCUSSION THREAD

**Discussion by rebokdev** [Music-Bot-for-Jitsi/Jimmi#146] [Q&A]
1. launch site
2. paste jitsi link
3. not working

ah, yes the web extension
where is it???
i cannot found it on repo

**Comment by tr1p0p** [comment]
Yeah, same question here.

**Reply by p-fruck** [reply]
Sorry, I must have missed the initial question. What you are looking for is located in the [Jimmi v1 release installation instructions](https://github.com/Music-Bot-for-Jitsi/Jimmi/tree/v1.0.0#installation). The current source code at the main branch is a POC that derived from a `Software Engineering II lesson` and is not going to be maintained because of the massive resource overhead. Once there is enough time to maintain this project I will pick up and continue the original version 1 release which is a lot less complex and resource intensive.

Note that the Browser extension should also work with Firefox since support for Manifest V3 has been added recently, but this hasn't been tested due to the lack of time.
